aws-auth-cognito/src/test/java/featureTest/utilities/APICaptorFactory.kt

```diff
@@ -60,7 +60,6 @@ class APICaptorFactory(
         val errorCaptor = slot<AuthException>()
         val actionCaptor = slot<Map<String, Any>>().apply {
             captured = emptyMap()
```

Comment:
mockk `isCaptured` is now a val that gets its value from `captured`: https://github.com/mockk/mockk/commit/d6585041993870c5802ab52b67cdac0c678807fe#diff-aee62ffced00343726c80334bdc58f9a200c6a84895a95240329a0f05aa0adc2R2651